composer.json

```diff
@@ -14,7 +14,8 @@
         "ext-libxml": "*",
         "ext-simplexml": "*",
         "giggsey/libphonenumber-for-php": "^7.0|^8.0",
-        "symfony/options-resolver": "^2.7|^3.0|^4.0"
+        "symfony/event-dispatcher-contracts": "^1.1|^2.2",
```

Comment:
Probably `psr/event-dispatcher` is better here as `Psr\EventDispatcher\EventDispatcherInterface` is required in Client.